blockchains/smartchain/assets/0x3E2ed75F718251Fd011312b9E33Bf10A4dbfda54/info.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "Many Worlds Token",
+    "website": "https://www.manyworldstoken.com",
+    "description": "Many Worlds Token is an innovative rewards token with the world's first dynamic tokenomics system. Built on the Binance Smart Chain, holders of MANY will be rewarded in BUSD for just holding the token.",
+    "explorer": "https://bscscan.com/token/0x3e2ed75f718251fd011312b9e33bf10a4dbfda54",
+    "type": "BEP20",
+    "symbol": "MANY",
+    "decimals": 9,
+    "status": "active",
+    "id": "0x3E2ed75F718251Fd011312b9E33Bf10A4dbfda54",
+    "links": [
+        {
+            "name": "github",
+            "url": "https://github.com/Many-Worlds-Token/"
+        },
+        {
+            "name": "twitter",
+            "url": "https://twitter.com/manyworldstoken"
+        },
+        {
+            "name": "reddit",
+            "url": "https://reddit.com/r/ManyWorldsToken/"

```

Comment:
https://reddit.com/ManyWorldsToken/ its correct url, please change

Reply:
Hey @cryptomanz, can you verify this I get a 404 with that URL

<img width="865" alt="Screenshot 2022-01-26 at 18 20 24" src="https://user-images.githubusercontent.com/50641262/151223153-d63879b4-07b7-42e9-8687-8f7704fd919f.png">